tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -191,7 +257,29 @@ public function filterResponseAccordingToNotificationSubject(string $subject): o
 	}
 
 	/**
-	 * @Then user :user should get a notification with subject :subject and message:
```

Comment:
i think this has been overwritten accidentallty. Should this be below the new function filterResponseByNotificationSubjectAndResource()

Reply:
I didn't remove this step. I added `have` word in this step so github shows this like this step has been removed nothing more @KarunAtreya